docs/TRACEZ_DESIGN.md

```diff
@@ -0,0 +1,175 @@
+# OpenTelemetry SDK Contrib - /tracez and /traceconfigz Design Doc
+
+This file contains information about the design choices for the OpenTelemetry /tracez and
+/traceconfigz zPages.
+
+## Introduction
+
+The OpenTelemetry zPages are a set of dynamically generated HTML pages that display trace and
```

Comment:
Suggested edit:

The OpenTelemetry zPages are a set of dynamically generated HTML pages that display trace and
metrics data for a running process: the /tracez zPage displays
trace information about running spans, sample span latencies, and sample error spans, while the /traceconfigz zPage is a web page that allows the users to change tracing parameters, such
as sampling probability and max number of attributes.